### ISSUE TITLE
UU/WCAG: Missing aria-value for TextArea-element

### ISSUE THREAD

**Issue by xmrsa** [Altinn/app-frontend-react#1538]
### Description of the bug

When I have a TextArea with only a title-text, Wave gives an error like this:

![image](https://github.com/Altinn/app-frontend-react/assets/42466346/77d180c0-5ddd-4572-9a23-fda8893a5745)

The mistake is that the aria-reference uses the "description". When I add a description it looks fine in Wave:
![image](https://github.com/Altinn/app-frontend-react/assets/42466346/2da76e49-8215-4595-bcf9-6851c5d1cc6d)

But I don't want a description her, I only want the title-element, like this:
  `   {
        "id": "TextArea-kommentarer-til-omsetningssiden",
        "type": "TextArea",
        "dataModelBindings": {
          "simpleBinding": "SkjemaData.KommentarSide.kommOmsetning"
        },
        "required": false,
        "textResourceBindings": {
          "title": "lang.kommentarer.til.spoersmaal.siden.title"
        }
      },`

### Steps To Reproduce

1. Make a TextArea-element with only "title"-element.
2. Run your app
3. Push Wave-button, and you will see for yourself.

**Comment by Febakke** [comment]
Thanks for reporting this! 🙌
I agree with you conclusion. Seems like a bug where the describedBy is not removed when there are no description. 
We should test if this only occurs with textarea or if the bug is present on other components as well.

**Comment by olemartinorg** [comment]
Related to:
- #1528

**Comment by xmrsa** [comment]
Noen planer om å fikse denne issuen snart? Det er litt irriterende å få feilmelding i Wave når det egentlig ikke er vi skjemautviklere som har gjort noe feil.

**Comment by olemartinorg** [comment]
@xmrsa Jeg kjenner ikke til noen umiddelbare planer, dessverre. Vi prøver å ta noen feilrapporter innimellom, spesielt om de er alvorlige, men det er mye å gjøre - for mye til at vi klarer å tilfredsstille alle behov, dessverre. Siden dere har kjøpt dere en egen prioriteringskø (eller et forbikjøringsfelt, om du vil) vil jeg anbefale å ta det videre internt hos dere og eventuelt flytte dette opp på lista om denne fiksen bør komme før andre ting.

**Comment by Magnusrm** [comment]
I can not reproduce this issue. The issue seems to have been fixed by using the Designsystemet Textarea component in the PR: https://github.com/Altinn/app-frontend-react/pull/2213
